.editorconfig

```diff
@@ -0,0 +1,6 @@
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 4
```

Comment:
not 3?

Reply:
Up to personal preference, I think this should be 2 or 4 but 3 seems like an odd number 🤔